arrow/api.py

```diff
@@ -22,7 +22,7 @@
 @overload
 def get(
     *,
-    locale: str = "en_us",
+    locale: str = "en-us",
```

Comment:
Should we maybe put this in the constants?

DEFAULT_LOCALE?

Reply:
Agreed. Done!